package OfficialConfig;

message OfficialJob
{
	required uint32		id 				= 1; //职位
	required uint32		coin_yield_add 	= 2; //
	required uint32		wood_yield_add 	= 3; //
	required uint32 	food_yield_add  = 4; //
	required uint32		iron_yield_add	= 5; //
	required uint32		pay				= 6; //俸禄	
}

message OfficialDepartment
{
	required uint32		 department = 1; //部门
	repeated OfficialJob job		= 2; //
}

message Official
{
	repeated OfficialDepartment official = 1;
}